clients/privacy-center/public/fides-js-components-demo.html

```diff
@@ -251,7 +256,17 @@ <h2>Consent JSON</h2>
       <pre id="consent-json"></pre>
 
       <h2>Consent Modal Link</h2>
-      <button id="fides-modal-link"></button>
+      <div id="render-link-here"></div>
+      <script>
+        setTimeout(() => {
+          const app = document.getElementById("render-link-here");
+          const button = document.createElement("button");
+          // Display the fides configured modal link label (if any).
+          button.textContent = Fides.getModalLinkLabel();
```

Comment:
hmmm....it's worse even. Since it delays the Overlay initialization and that's where the link gets bound to the click handler and more importantly the classname to make it show up...the link doesn't show up until after the embed is found. Definitely a problem.